docs/Users_Guide/use-cases/lulc/nsf-ncar.rst

```diff
@@ -0,0 +1,144 @@
+
+  .. dropdown:: Define Environment Variables
+
+    We will be using environment variables throughout this exercise to ensure consistent file paths and resource names. Copy and paste the definitions below into your shell before proceeding::
+
+        IWRF_WORK_DIR=${SCRATCH}/iwrf_work
+        LOCAL_OUTPUT_DIR=${IWRF_WORK_DIR}/metplus_out
+        export APPTAINER_TMPDIR=${TMPDIR}
+
+    Any time you open a new shell session on the HPC system, you will need to reload the apptainer module, switch shells, if needed, and redefine these variables before executing the commands that follow.
+
+  .. dropdown:: Create Working Directories
+
+    The METplus verification process requires specific directory structures to organize input data, configuration files, and output results. Create the main working directory in your scratch space::
+
+        mkdir -p ${IWRF_WORK_DIR}
+
+    Create a directory to store the METplus verification output::
+
+        mkdir -p ${LOCAL_OUTPUT_DIR}
+
+    Create a directory for temporary Apptainer files. The $TMPDIR variable is automatically set on NCAR HPC systems to an appropriate temporary storage location::
+
+        mkdir -p ${APPTAINER_TMPDIR}
+
+  .. dropdown:: Download Configuration Files
+
+    METplus requires configuration files to direct its verification behavior. These are available in the I-WRF GitHub repository. Clone the repository to access the LULC use case configuration::
+
+        git clone https://github.com/NCAR/i-wrf ${IWRF_WORK_DIR}/i-wrf
+
+    This creates a local copy of all I-WRF configuration files, including the METplus settings needed for the LULC verification workflow.
+
+  .. dropdown:: Get the METplus and Data Container Images
+
+    As mentioned above, the METplus software is provided as a containerized image that will run using Apptainer on your NCAR HPC system. Apptainer (formerly Singularity) is the preferred containerization technology on HPC systems, as it provides secure container execution without requiring root privileges. Unlike cloud environments that use Docker directly, NCAR HPC systems use Apptainer to run containerized applications.
+
+    The METplus image contains all the necessary software and dependencies to perform verification of I-WRF LULC simulation output. You can "pull" (download) the METplus image from the container registry to your HPC system's storage.
+
+    You must pull the METplus software container and the input data containers that contain the observational and WRF simulation data for the LULC use case::
+
+        apptainer pull ${IWRF_WORK_DIR}/iwrf-metplus.sif docker://ncar/iwrf-metplus:latest
+        apptainer pull ${IWRF_WORK_DIR}/data-lulc-input-obs.sif oras://registry-1.docker.io/ncar/iwrf-data:lulc-input-obs-d03.apptainer
+        apptainer pull ${IWRF_WORK_DIR}/data-lulc-input-wrf.sif oras://registry-1.docker.io/iwrf-data:lulc-input-wrf-d03.apptainer
```

Comment:
Sorry! I think I introduced this typo. Need to fix the `oras` path to add in the missing `ncar`.
```suggestion
        apptainer pull ${IWRF_WORK_DIR}/data-lulc-input-wrf.sif oras://registry-1.docker.io/ncar/iwrf-data:lulc-input-wrf-d03.apptainer
```